rakefile.rb

```diff
@@ -1,9 +1,9 @@
 require 'albacore'
 
-msbuild_command = [
-  "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe",
-  "C:/Program Files (x86)/MSBuild/12.0/Bin/MSBuild.exe"
-].find(lambda{ raise "MSBuild not found"}) { |msbuild| File.file?(msbuild) }
+msbuild_command = "C:/Program Files (x86)/MSBuild/14.0/Bin/MSBuild.exe"
+if !File.file?(msbuild_command)
+  raise "MSBuild not found"
+end

```

Comment:
And I was so proud of having learned the fancy 'find in array or blow up' construction. Ah well.